src/firestore/api-sort.ts

```diff
@@ -225,6 +230,19 @@ function compareArrayConfig(a?: API.ArrayConfig, b?: API.ArrayConfig): number {
   return ARRAY_CONFIG_SEQUENCE.indexOf(a) - ARRAY_CONFIG_SEQUENCE.indexOf(b);
 }
 
+function compareVectorConfig(a?: API.VectorConfig, b?: API.VectorConfig): number {
+  if (a === undefined) {
```

Comment:
I'd prefer using `!a` rather than `a === undefined` just in case an errant `null` gets in here.